pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[build-system]
+requires = ["scikit-build-core", "numpy", "setuptools_scm>=8"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "adios2"
+dynamic = ["version"]
+authors = [
+  { name="Chuck Atkins", email="chuck.atkins@kitware.com" },
+  { name="Greg S. Eisenhauer", email="eisen@cc.gatech.edu" },
+  { name="William F. Godoy", email="godoywf@ornl.gov" },
+  { name="Junmin Gu", email="jgu@lbl.gov" },
+  { name="Norbert Podhorszki", email="pnorbert@ornl.gov" },
+  { name="Ruonan (Jason) Wang", email="wangr1@ornl.gov" },
```

Comment:
This has to be updated

Reply:
I took these names right from the copyright statement.  If they need to be updated, just need to let me know what names should be there.

Reply:
hmm you are right we have to update the copyright.txt as well, lets have that conversation there with the rest of the team.

Reply:
#3965

Reply:
Thanks, I've done this.